views/js/controller/runner/runner.js

```diff
@@ -89,6 +91,8 @@ define([
                 plugins : config && config.providers && Object.keys(config.providers.plugins)
             });
 
+            $('.delivery-scope').attr({dir: locale.getLanguageDirection(context.locale)});
```

Comment:
**thought:** Looks good if we consider not only the test runner but the whole page.
However, I'd rather place this aside with the loading bar start for better consistency, [around line 175](https://github.com/oat-sa/extension-tao-testqti/pull/2020/files#diff-9c3d580247a181bfe1ce77bb1052b6e90941ac5231603179fb7bea049cc0df8fR175)